Documentation/workflow/OfficalAndDailyBuilds.md

```diff
@@ -12,11 +12,36 @@ and you can expect to see new versions roughly three months.   However it is als
 Team publishes **daily builds** of all sorts of packages including those built by the CoreCLR and CoreFX 
 repositories.  You can see what is available from
 
- * <https://dotnet.myget.org/gallery/dotnet-core>, and in particular you can see the builds of CoreCLR at 
- * <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.Runtime.CoreCLR>.   
- 
+ * <https://dotnet.myget.org/gallery/dotnet-core>, and in particular you can see the builds of 
+ * CoreCLR at <https://dotnet.myget.org/feed/dotnet-core/package/nuget/Microsoft.NETCore.Runtime.CoreCLR>
```

Comment:
NIT: is this extra bullet point intentional? I ask since it seems like it is a continuation from the first one